docs/builders/creating_themes.rst

```diff
@@ -80,7 +80,9 @@ Head Components
 
 .. vale on
 
-At present, Mautic won't process the ``<mj-head>`` tags. None of the ``<mj-attribuites>`` run, so you have to do all styling inline.
+Mautic processes most of the ``<mj-head>`` components. ``<mj-attributes>`` do not run.
+
+**Tested elements** are: mj-breakpoint, mj-font, mj-html-attributes, mj-style, mj-title, mj-preview 
```

Comment:
```suggestion
**Tested elements** include: ``mj-breakpoint``, ``mj-font``, ``mj-html-attributes``, ``mj-style``, ``mj-title``, and ``mj-preview``.
```

Let's make sure these are all enclosed in inline code blocks. This will prevent the Vale warning about HTML and also makes it clear we're referring to code-related terms.

Reply:
Committed changes